web/src/components/Select.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Select = ({ name, options, label, hideLabel, wrapperClass, ...rest }) => (
+  <div className={wrapperClass || 'mb2'}>
+    <label
+      htmlFor={name}
+      className={hideLabel ? 'sr-only' : 'block mb-tiny bold'}
+    >
+      {label}
+    </label>
+    <select className="m0 select" {...rest}>
```

Comment:
Should this have an `id=${name}` so the label has something to relate to?

Reply:
good catch, yes it does, fixing and merging, thx!